package.json

```diff
@@ -4,10 +4,25 @@
     "mustache": "^4.2.0"
   },
   "devDependencies": {
-    "husky": "^8.0.0",
-    "pretty-quick": "^3.1.3"
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
+    "eslint": "^8.23.1",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "husky": "^8.0.1",
+    "pretty-quick": "^3.1.3",
+    "lint-staged": "^13.0.3",
+    "prettier": "^2.7.1"
+  },
+  "lint-staged": {
+    "*.ts": "eslint --cache --fix",
+    "*.{ts,css,md}": "prettier --write"
   },
   "scripts": {
+    "prettier": "prettier --loglevel silent --write \"**/*.{md,jsx,tsx,ts,json,html,css,js,yml}\"",
```

Comment:
should the last type read `yaml`?

Reply:
Hmm, I actually think I should remove this since we have not been using prettier on yaml. But, if we wanted it, we would probably want to add `.yml` and `.yaml` since they are used interchangeably.